src/common/framework/util/util.ts

```diff
@@ -48,6 +48,14 @@ export function raceWithRejectOnTimeout<T>(p: Promise<T>, ms: number, msg: strin
   return Promise.race([p, rejectOnTimeout(ms, msg)]);
 }
 
+export function sortObjectByKey(v: { [k: string]: unknown }): object {
```

Comment:
nit: return type  could be more specific
```suggestion
export function sortObjectByKey(v: { [k: string]: unknown }): { [k: string]: unknown } {
```